lib/fund_america/investment.rb

```diff
@@ -20,7 +20,8 @@ def create(options)
       # Usage: FundAmerica::Investment.update(investment_id, options)
       # Output: Updates an investment
       def update(investment_id, options)
-        API::request(:patch, "investments/#{investment_id}", options)
+        end_point_url = FundAmerica.base_uri + "#{FundAmerica.mode == 'sandbox' ? 'test_mode/' : ''}" + "investments/#{investment_id}"
```

Comment:
Here is a bug. I'm not sure if we will be using this endpoint in production mode, but it uses wrong url in such case. Example:
```
https://apps.fundamerica.com/api/https://apps.fundamerica.com/api/investments/ID
```